### DISCUSSION TITLE
Fail to get master client from cache when start Backend.

### DISCUSSION THREAD

**Discussion by uniorder** [apache/doris#13946] [A - General / Q&A]
Error:

W1103 16:25:25.688463  9901 utils.cpp:90] fail to get master client from cache. host=127.0.0.1, port=9020, code=THRIFT_RPC_ERROR
W1103 16:25:25.688509  9901 task_worker_pool.cpp:1598] report TASK failed. status: Internal error: Fail to get master client from cache, master host: 127.0.0.1, port:9020

**Comment by uniorder** [comment]
Resolved, that is my configuration wrong.

priority_networks on be side is fe ip address.

**Reply by tilepotter** [reply]
How did you solve it? How do you need to configure priority_networks correctly?

**Reply by ixzc** [reply]
refer this : https://doris.apache.org/zh-CN/docs/dev/get-starting/quick-start
